docs/user_guide/1_setting_up_the_hand.md

```diff
@@ -99,84 +100,41 @@ see all joints of the hand move slightly on power up or reset or power up.
 
   ROS Kinetic (Recommended):
   ```bash
-  $ bash <(curl -Ls bit.ly/run-aurora) docker_deploy product=hand_e ethercat_interface=[EtherCAT interface ID] config_branch=[sr_config_branch]
+  $ bash <(curl -Ls bit.ly/run-aurora) server_and_nuc_deploy --read-input docker_username --read-secure docker_password --limit 'server'
   ```
-  Examples:
-  For Interface ID ```ens0s25``` and sr_config_branch ```shadow_12345```
-  ```bash
-  $ bash <(curl -Ls bit.ly/run-aurora) docker_deploy product=hand_e ethercat_interface=ens0s25 config_branch=shadow_12345
-  ```  
   Same as above but with ROS logs upload enabled
   ```bash
-  $ bash <(curl -Ls bit.ly/run-aurora) docker_deploy product=hand_e ethercat_interface=ens0s25 config_branch=shadow_12345 use_aws=true
+  $ bash <(curl -Ls bit.ly/run-aurora) server_and_nuc_deploy --read-input docker_username --read-secure docker_password --limit 'server' use_aws=true
   ```  
-  
-  If you have an Nvidia graphics card, you can add nvidia_docker=true to use nvidia-docker
-
-  ROS Indigo:
+  If you do not have an Nvidia graphics card, you can add nvidia_docker=false to use nvidia-docker (`true` is out default), i.e.:
   ```bash
-  $ bash <(curl -Ls bit.ly/run-aurora) docker_deploy product=hand_e ethercat_interface=[EtherCAT interface ID] config_branch=[sr_config_branch] tag=indigo-release
+  $ bash <(curl -Ls bit.ly/run-aurora) server_and_nuc_deploy --read-input docker_username --read-secure docker_password --limit 'server' use_aws=true nvidia_docker=false
   ```
-  Examples:
-  For Interface ID ```ens0s25``` and sr_config_branch ```shadow_12345```
-  ```bash
-  $ bash <(curl -Ls bit.ly/run-aurora) docker_deploy product=hand_e ethercat_interface=ens0s25 config_branch=shadow_12345 tag=indigo-release
-  ```  
-  Same as above but with ROS logs upload enabled
-  ```bash
-  $ bash <(curl -Ls bit.ly/run-aurora) docker_deploy product=hand_e ethercat_interface=ens0s25 config_branch=shadow_12345 tag=indigo-release use_aws=true
-  ```  
-
-  You can also add reinstall=true true in case you want to reinstall the docker image and container. When it finishes it will show if it was successful or not
-  and it will create five desktop icons on your desktop that you can double-click to launch the hand container, save the log files from the active containers to your desktop and perform various actions on the hand (open, close and demo).
-  The icon that launches the hand looks like this:
 
-  ![desktop_icon](../img/desktop_icon.png)
-
-  And for saving the logs:
-
-  ![log_icon](../img/log_icon.png)
+  You can also add `reinstall=true` in case you want to reinstall the docker image and container. When it finishes it will show if it was successful or not
+  and will create desktop icons on your desktop that you can double-click to launch the hand container, save the log files from the active containers to your desktop and perform various actions on the hand (open, close and demo).
+  The icon look like this:
```

Comment:
The icons will look like this:

Reply:
Also have a bigger screenshot which shows what's inside the folders, and also explain what each icon does